lib/cc/engine/analyzers/php/ast.rb

```diff
@@ -1,51 +0,0 @@
-# frozen_string_literal: true
-
-require "cc/engine/analyzers/php/nodes"
-
-module CC
-  module Engine
-    module Analyzers
-      module Php
```

Comment:
I thnk we usually capitalize acronym class/module names. So maybe this should be `PHP`?